src/Microsoft.ApplicationInsights.AspNet/project.json

```diff
@@ -25,7 +25,15 @@
     },
 
     "frameworks": {
-        "dnx451": { },
+      "dnx451": {
+        "compilationOptions": {
+          "define": [ "net451" ]
+        },
+        "dependencies": {
+          "Microsoft.ApplicationInsights.PerfCounterCollector": "2.0.0-*",
```

Comment:
Use specific version as you use for "Microsoft.ApplicationInsights" instead of star. Alternatively change it in both places to star